deploy/development/deployment.yaml

```diff
@@ -61,14 +61,56 @@ spec:
           value: "true"
         - name: KUBERNETES_DEPLOYMENT
           value: "true"
+        - name: NOMIS_OAUTH_HOST
+          value: "https://sign-in-dev.hmpps.service.justice.gov.uk"
+        - name: NOMIS_STAFF_SLOT_AVAILABILITY_ENABLED
+          value: "true"
+        - name: PRISON_API_HOST
+          value: "https://prison-api-dev.prison.service.justice.gov.uk"
         - name:  PRISON_VISITS_API
           value: "https://prison-visits-booking-staff-dev.apps.live.cloud-platform.service.justice.gov.uk/"
+        - name: PUBLIC_PRISONS_WITH_SLOT_AVAILABILITY
+          value: >
+            Bedford,
+            Feltham (Young People 15-18 only),
+            Feltham (Young Adults 18-21 only),
+            High Down,
+            Hull,
+            Medway Secure Training Centre,
+            North Sea Camp,
+            Nottingham,
+            Swaleside,
+            Whitemoor,
+        - name: PUBLIC_SERVICE_URL
```

Comment:
instead of using this new `PUBLIC_SERVICE_URL` - could we not use the `SERVICE_URL` on line 102 - or vice-versa

Reply:
In terms of using PUBLIC_SERVICE_URL, STAFF_SERVICE_URL or SERVICE_URL, I blindly imported whichever from the staff app into the public app as not to interfere.

I will investigate whether there is every any difference in the various environments and unify under one.

Reply:
if in the Public app `SERVICE_URL` is not used - shouldn't we refrain from adding it to the deployment file as a new environment variable

Reply:
This above analysis was done against main repo for public and staff.  When staff bits are merged into public in VB-3457 all  3 env vars are used and seem to be set to difference things.

Reply:
ahh ok, makes sense that it's included then